includes/features/onboarding/class-wc-admin-onboarding.php

```diff
@@ -51,13 +51,64 @@ public function should_show_profiler() {
 		return $is_completed || $is_skipped ? false : true;
 	}
 
+	/**
+	 * Get a list of allowed product types for the onboarding wizard.
+	 *
+	 * @return array
+	 */
+	public static function get_allowed_product_types() {
+		return apply_filters(
+			'woocommerce_admin_onboarding_product_types',
+			array(
+				'physical'      => array(
+					'label'       => __( 'Physical products', 'woocommerce-admin' ),
+					'description' => __( 'Products you ship to customers.', 'woocommerce-admin' ),
+				),
+				'downloads'     => array(
+					'label'       => __( 'Downloads', 'woocommerce-admin' ),
+					'description' => __( 'Virtual products that customers download.', 'woocommerce-admin' ),
+				),
+				'subscriptions' => array(
+					'label'       => __( 'Subscriptions — $199 per year', 'woocommerce-admin' ),
+					'description' => __( 'Products with recurring payment.', 'woocommerce-admin' ),
+					'more_url'    => __( 'https://woocommerce.com/products/woocommerce-subscriptions/', 'woocommerce-admin' ),
+				),
+				'memberships'   => array(
+					'label'       => __( 'Memberships — $199 per year', 'woocommerce-admin' ),
+					'description' => __( 'Restrict content to customer groups.', 'woocommerce-admin' ),
+					'more_url'    => __( 'https://woocommerce.com/products/woocommerce-memberships/', 'woocommerce-admin' ),
+				),
+				'composite'     => array(
+					'label'       => __( 'Composite Products — $199 per year', 'woocommerce-admin' ),
+					'description' => __( 'Kits with configurable components.', 'woocommerce-admin' ),
+					'more_url'    => __( 'https://woocommerce.com/products/composite-products/', 'woocommerce-admin' ),
+				),
+				'spaces'        => array(
+					'label'       => __( 'Spaces — $199 per year', 'woocommerce-admin' ),
+					'description' => __( 'Sell access to spaces, e.g. hotel rooms.', 'woocommerce-admin' ),
+					// @todo Need a link for this product information.
+					'more_url'    => __( '#', 'woocommerce-admin' ),
```

Comment:
`spaces` could link to https://woocommerce.com/products/woocommerce-accommodation-bookings/ -- though i wonder if that would be confusing because its listed as free (but requires the bookings plugin). cc @jameskoster

Reply:
I do think this will be a bit confusing.  While we can list bookings for both, it seems like checking both would imply the need to pay the fee twice, which wouldn't be the actual case.

@jameskoster Any thoughts on what to do here?